src/main/treeDataProviders/utils/utils.ts

```diff
@@ -14,6 +14,14 @@ export class Utils {
         return path.substring(path.lastIndexOf(pathUtils.sep) + 1);
     }
 
+    public static tryRelativePath(full: string, potentialParent?: string): string {
+        if (potentialParent && full.startsWith(potentialParent)) {
+            let localPath: string = full.substring(potentialParent.length + 1);
+            return './' + localPath;
```

Comment:
Can we change it to this?
```suggestion
            return full.substring(potentialParent.length + 1);
```

Explanation - I'm not sure that "./" would work on Windows. Do we really need it?

Reply:
removed the translation to relative.